modules/ROOT/pages/mmp-concept.adoc

```diff
@@ -494,22 +559,22 @@ The example is using placeholder values. Configure each value with your local Mu
 +
 [source,console,linenums]
 ----
-> mvn clean package deploy -DmuleDeploy
+$ mvn clean package deploy -DmuleDeploy
 ----
 
 === Standalone Deployment Reference
 
 [%header,cols="30,70"]
 |===
 |Parameter | Description
-|standaloneDeployment | Top-Level Element.
-| muleVersion | The Mule runtime version running in your local machine instance. +
+|`standaloneDeployment` | Top-Level Element.
+| `muleVersion` | The Mule runtime version running in your local machine instance. +
 If this value does not match the runtime version running in your deployment target, the plugin raises an exception.
 
 The Mule Maven Plugin does not download a Mule runtime if these values don't match.
-| muleHome | The location of the Mule Runtime instance in your local machine.
-include::mule-runtime:ROOT:page$mmp-concept.adoc[tag=deploymentTimeoutParameterDescription]
-include::mule-runtime:ROOT:page$mmp-concept.adoc[tag=skipParameterDescription]
+| `muleHome` | The location of the Mule Runtime instance in your local machine.
```

Comment:
Mule Runtime > Mule runtime instance
So you keep the same terminology for all Mule runtime in the doc.